src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int radius = random.nextInt(1,10);
+        int sideA = random.nextInt(1,10);
+        int sideB = random.nextInt(1,10);
+        int height = random.nextInt(1, 10);
+        int index = random.nextInt(1, 5);
```

Comment:
Better to extract magic numbers to constant with proper naming
```suggestion
        int index = random.nextInt(1, NUMBER_OF_FIGURES);
```